app/services/orderService.js

```diff
@@ -82,9 +80,69 @@ class OrderService {
         strategy.now_buy_times = now_buy_times;
         await strategy.save();
 
-        return { order_id: order_id }
+        return { order_id: inOrder.order_id }
     }
 
+    async sellOrder(strategy_id) {
+        const item = await Strategy.findById(strategy_id);
+        // Access the current price
+        const exchange = new (ccxt)[item.exchange]({
+            apiKey: item.key,
+            secret: item.secret,
+            timeout: 5000,
+        });
+        
+        const tickerRes =  await exchange.fetchTicker(item.symbol);
+        const sell_price = tickerRes.bid;
+
+        // Check if the profit reach the setting take-profit rate
```

Comment:
From my perspective, this piece of code should be encapsulated.
For instance, you can build a class named Strategy that has the methods called check_profit, etc.
A good encapsulation could make the code more readable.

Although the code is for machine to execute, it's also for human to read.